src/com/jeantessier/classreader/ClassfileLoaderEventSource.java

```diff
@@ -128,6 +128,13 @@ protected void load(String filename, InputStream in) {
 
             case CLASS:
                 Logger.getLogger(getClass()).debug("CLASS \"" + filename + "\"");
+                
+                // Ignore Java 9 Jigsaw module info class (fix issue #4) 
+                if ("module-info.class".equals(filename)) {
```

Comment:
I think it's better to widen the scope of the `catch` clause on line 142 than to arbitrarily exclude a file.  This way, it will fail gracefully whenever it encounters new bytecode bits.  The `try ... catch` statement already intended to swallow `IOException`, we can just broaden the scope a little.

I already put it on `master`.

Reply:
You know the code better than I doo, I'm perfectly fine with your fix, thanks!